ddtrace/tracer/sampler.go

```diff
@@ -412,11 +412,12 @@ func (sr *SamplingRule) MarshalJSON() ([]byte, error) {
 type rateLimiter struct {
 	limiter *rate.Limiter
 
-	mu       sync.Mutex // guards below fields
-	prevTime time.Time  // time at which prevRate was set
-	prevRate float64    // previous second's rate.
-	allowed  int        // number of spans allowed in the current period
-	seen     int        // number of spans seen in the current period
+	mu          sync.Mutex // guards below fields
+	prevTime    time.Time  // time at which prevRate was set
```

Comment:
`prevRate` does not exist anymore. 
```suggestion
	prevTime    time.Time  // time at which prevAllowed and prevSeen were set
```